### PR TITLE
added Edward Wu to people

### PR DESCRIPTION
**Pull request by wuedward**

### CODE REVIEW

**Comment by dblock** [comment]
Good, do you think you can try to _rebase_ this against master and _squash_ the commits before I merge it?

**Comment by wuedward** [comment]
I ran "git rebase -i" and got a "noop". Does this mean I have done the rebase and squash correctly?

**Comment by dblock** [comment]
Nope, this will need a bit more work. Here it should look like 1 commit, not 2.

The easiest is to first sync master to upstream, so:

```
# go to master
git checkout master

# create a new temporary branch
git checkout -b tmp

# go back to master
git checkout master

# rewind a few commits, lose changes (they are saved on tmp)
git reset HEAD~10 --hard

# update from upstream
git pull upstream master

# push master to github
git push origin master -f
```

Now we'll apply the change to a new branch.

```
# create a new branch
git checkout -b student-info

# merge from tmp, without the history
git merge tmp --squash

# examine changes, commit
git add .
git commit ...

# push to github
git push origin student-info
```

Now make the pull request.

File: people/edward-wu.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Edward Wu
+--------
+![](photos/wuedward.jpg)
```
Comment (dblock) [inline people/edward-wu.md]: Please name the file the same as your name, so `edward-wu.jpg`.

**Pull request closed without merging**